cmd/root.go

```diff
@@ -131,13 +131,22 @@ func displayDB(rows *sql.Rows) error {
 		if err != nil {
 			return err
 		}
-	//TODO: switch between table and csv dependent on num columns(suggested num for table 5<=
 	default:
-		err := tableDisplay(rows)
+		columns, err := rows.Columns()
 		if err != nil {
 			return err
 		}
-
+		if shouldUseTable(len(columns)) {
```

Comment:
The build error you were getting was because rows.Columns() can return an error. len(columns) will just return an int. 

What I had meant when I said a helper function was something like
`
func shouldUseTable(rows *sql.rows) bool{
err := rows.columns()
if err != nil {
fmt.println(err)
os.Exit(1) <-- or don't exit and default to something
}
return len(columns)<5
}
`

What you have here is good but would be cleaner to just have if len(columns)<=5

Reply:
Oh okay, I'll do that now. Sorry if I'm dragging this too long